zcash_client_sqlite/src/wallet/init/migrations/v_transactions_net.rs

```diff
@@ -0,0 +1,318 @@
+//! Migration that fixes a bug in v_transactions that caused the change to be incorrectly ignored
+//! as received value.
+use std::collections::HashSet;
+
+use rusqlite;
+use schemer;
+use schemer_rusqlite::RusqliteMigration;
+use uuid::Uuid;
+
+use super::add_transaction_views;
+use crate::wallet::init::WalletMigrationError;
+
+pub(super) const MIGRATION_ID: Uuid = Uuid::from_fields(
+    0x2aa4d24f,
+    0x51aa,
+    0x4a4c,
+    b"\x8d\x9b\xe5\xb8\xa7\x62\x86\x5f",
+);
+
+pub(crate) struct Migration;
+
+impl schemer::Migration for Migration {
+    fn id(&self) -> Uuid {
+        MIGRATION_ID
+    }
+
+    fn dependencies(&self) -> HashSet<Uuid> {
+        [add_transaction_views::MIGRATION_ID].into_iter().collect()
+    }
+
+    fn description(&self) -> &'static str {
+        "Fix transaction views to correctly handle double-entry accounting for change."
+    }
+}
+
+impl RusqliteMigration for Migration {
+    type Error = WalletMigrationError;
+
+    fn up(&self, transaction: &rusqlite::Transaction) -> Result<(), WalletMigrationError> {
+        transaction.execute_batch(
+            "INSERT INTO sent_notes (tx, output_pool, output_index, from_account, to_account, value)
+             SELECT tx, 2, output_index, account, account, value 
```

Comment:
Can we avoid actually hard-coding `2` here, and instead use an existing constant?